scripts/create_pelorus_operator

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+#
+# Copyright Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+OPERATOR_PROJECT_NAME=pelorus-operator
+OPERATOR_ORG_NAME=migtools
+OPERATOR_PROJECT_DOMAIN=pelorus.konveyor.io
+
+# Get the full absolute path to the script. Needed while calling the script
+# via various partial paths or sourcing the file from shell
+# BASH_SOURCE[0] is safer then $0 when sourcing.
+# We enter the dirname of the invoked script and then get the current
+# path of the script using pwd
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
Would be good to change this script from Bash to Python to increase readability and maintainability? (since it is already tested, this can be a future task)

Reply:
We can do that but as you wrote, later., however I am in favor of shell when it comes to calling other CLIs as in this script (operator-sdk).